src/collide_vss.cpp

```diff
@@ -765,6 +772,105 @@ void CollideVSS::EEXCHANGE_ReactingEDisposal(Particle::OnePart *ip,
 
 /* ---------------------------------------------------------------------- */
 
+int CollideVSS::split(Particle::OnePart *&ip,
```

Comment:
Is `CollideVSS::split` unique to VSS, or could it be moved to `Collide` instead?